tests/vibe.http.2080/source/app.d

```diff
@@ -0,0 +1,32 @@
+/++ dub.sdl:
+	dependency "vibe-d" version="~>0.8.3-alpha.4"
```

Comment:
Shouldn't this be `path` too? (we probably should allow single files in the test suite too)

Reply:
True and true!